src/IbuMethods.cpp

```diff
@@ -61,3 +69,52 @@ double IbuMethods::rager(double AArating, double hops_grams, double finalVolume_
 
    return (hops_grams*utilization*AArating*1000)/(finalVolume_liters*(1+gravityFactor));
 }
+
+
+double IbuMethods::noonan(double AArating, double hops_grams, double finalVolume_liters, double wort_grav, double minutes)
+{
+    double gramsPerOz = 28.3495231;
+    double litersPerFiveGallons = 18.9270589; // US gallons
+    double volumeFactor = litersPerFiveGallons / finalVolume_liters;
+    double hopsFactor = hops_grams/ gramsPerOz;
+    double polynomialFactors[] = { 0.7000029428,
+                                  -0.08868853463,
+                                   0.02720809386,
+                                  -0.002340415323,
+                                   0.00009925450081,
+                                  -0.000002102006144,
+                                   0.00000002132644293,
+                                  -0.00000000008229488217
+                                 };
+
+    static Polynomial p = Polynomial(polynomialFactors, 7);
```

Comment:
You can initialize `p` without the auxiliary array `polynomialFactors`:

    static const Polynomial p( Polynomial() << 0.7... << -.08... << ... << -0.0000008 );

See [Algorithms.cpp:25](https://github.com/Brewtarget/brewtarget/blob/master/src/Algorithms.cpp#L25) for an example.